clustoapi/server.py

```diff
@@ -356,6 +356,7 @@ def get_from_pools():
     drivers = bottle.request.params.getall('driver')
     children = bottle.request.params.get('children', default=True, type=bool)
     mode = bottle.request.headers.get('Clusto-Mode', default='compact')
+    minify = bottle.request.headers.get('Clusto-Minify', default='False')
```

Comment:
Discussed in IRC, headers are string-only and if casted as a bool, always return True.